lib/user.cpp

```diff
@@ -47,8 +50,25 @@ class User::Private
             return Avatar(move(url));
         }
 
+        int makeHue(qreal hueF)
+        {
+            return static_cast<int>(hueF*360);
+        }
```

Comment:
This calculation is simple enough to not cache. Just do it on the fly in `User::hue()`.